auto-update-pip-packages/README.md

```diff
@@ -6,9 +6,9 @@
 
 Install the cron job. As root:
 
-```bash
-# cp cron /etc/cron.d/openfisca-auto-update-pip-packages
-# systemctl restart cron
+```sh
```

Comment:
I was not sure `bash` would be treated the same way, but it seems to be. Changing `bash` to `sh`  was probably not very usefull ;)